docs/podman-create.1.md

```diff
@@ -933,6 +933,19 @@ Precedence Order:
 
 	   **--env** : Any environment variables specified will override previous settings.
 
+Create containers and set the environment ending with a --*-- and a *****
```

Comment:
hmm, the `__*__` is not resolving here like it does at line 256.  Can you cut the one from line 256 and paste it in here?